src/main/java/fr/insalyon/creatis/gasw/parser/GaswParser.java

```diff
@@ -202,15 +206,24 @@ public void startElement(String uri, String localName, String qName, Attributes
                 throw new SAXException("<template> tags are just allowed inside <output> tags.");
             }
             String value = getAttributeValue(attributes, "value", "No template value defined.");
+            String stripExtns = getAttributeStripExtension(attributes, "strip-extension");
```

Comment:
Maybe we should use `strip-extensions` with a plural as there could be several ?